shared_model/backend/protobuf/impl/block.cpp

```diff
@@ -131,6 +132,7 @@ namespace shared_model {
     }
 
     interface::types::TransactionsNumberType Block::txsNumber() const {
+      // return boost::size(transactions());
```

Comment:
```suggestion
```

Reply:
This function is never used in production code.

Reply:
removing

Reply:
removed